finalPrep/modZeroHero.js

```diff
@@ -1,52 +1,93 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName = "J-Dawg";
+var specialAbility = "transports herself to another location in entire world with a sneeze";
 
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeting = "What's up! I'm J-Dawg!";
```

Comment:
Could you write this line to use `concatenation` so that "J-Dawg" is not hard-coded, rather it comes from the `heroName` variable?